azext_edge/tests/edge/support/test_support_unit.py

```diff
@@ -382,10 +376,7 @@ def test_create_bundle(
             )
 
             assert_list_replica_sets(
-                mocked_client,
-                mocked_zipfile,
-                label_selector=DATA_PROCESSOR_LABEL,
-                resource_api=DATA_PROCESSOR_API_V1
+                mocked_client, mocked_zipfile, label_selector=DATA_PROCESSOR_LABEL, resource_api=DATA_PROCESSOR_API_V1
```

Comment:
nit: **,** after resource_api=DATA_PROCESSOR_API_V1 since you added elsewhere?😋

Reply:
done